mutt2task.py

```diff
@@ -76,10 +93,27 @@
  
 # customize your own taskwarrior line
 # use `message' to add the subject
-res = Popen(['task', 'add', 'pri:L', '+mail', '--', message], stdout=PIPE)
-match = re.match("Created task (\d+)\.", res.stdout.read())
+if message == "None":
+    message = "E-Mail import: no subject specified."
+else:
+    message = "E-Mail subject: " + message
+
+res = Popen(['task', 'add', 'pri:L', '+email', '--', message], stdout=PIPE)
```

Comment:
Not sure if it ok to change tag name. But may be you are right here.

Reply:
It is working perfectly here. I am unaware of any side effects the tag might have. If there is a special requirement for any particular functionality to be available/triggered we can easily adapt it.

The reason I changed it to `email` is I like to see at a glance that this task got imported from an email. It makes filtering easier, just call `task +email all` and there they are.

Reply:
Only side effect is that I used to old tag. But I feel ok if you make it more precise.

Reply:
Of course, if you are used to it. OTOH, it is more concise, yes.